api/src/controllers/users.js

```diff
@@ -284,5 +281,20 @@ module.exports = {
         serverUtils.error(error, req, res);
       }
     },
+  },
+  v3: {
+    create: async (req, res) => {
+      try {
+        await auth.check(req, ['can_edit', 'can_create_users']);
+
+        const response = await users.createMultiFacilityUser(req.body, getAppUrl(req));
```

Comment:
So this endpoint will throw an error if the role lacks the multi-facility permission but multiple facilities are sent?

Reply:
Yes, if roles lack multi-facility permission but receive a user with 1 facility, then it's successful. If it receives many facilities then throws an error

Reply:
I think this is adds a bit of overhead, because we don't have an authorization library available to us in the users shared library. It's doable, but adds some refactors.

Reply:
Done